spec/gitlab_merge_request_hook_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'gitlab_merge_request_hook'
+
+describe GitlabMergeRequestHook do
+  let(:changes)         { "123456 789012 refs/heads/#{branch_name}" }
+  let(:config)          { GitlabConfig.new }
+  let(:repository_path) { "/home/git/repositories" }
+  let(:repo_name)       { 'dzaporozhets/gitlab-ci' }
+  let(:repo_path)       { File.join(repository_path, repo_name) + ".git" }
+  let(:hook)            { described_class.new(repo_path, config, changes) }
+
+  before do
+    GitlabConfig.any_instance.stub(repos_path: repository_path)
+    GitlabConfig.any_instance.stub(gitlab_url: "http://mygitlab.com")
+  end
+
+  describe "#exec" do
+    subject { hook.exec }
+
+    context "pushing on master" do
+      let(:branch_name) { "master" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.